en/docs/build-and-run/tryit-tools/graphql-tool.md

```diff
@@ -1 +1,56 @@
-# GraphQL tool
+# Try GraphQL services with Ballerina
+GraphQL is an open-source data query and manipulation language for APIs. While you develop an GraphQL service, you need to try it and debug it to check how it works. The Ballerina VS Code plugin provides the GraphQL Try it view, which gives the ability to try GraphQL services within VS Code instead of using any third-party software.
```

Comment:
```suggestion
GraphQL is an open-source data querying and manipulation language for APIs. While you develop a GraphQL service, you need to try it and debug it to check how it works. The Ballerina VS Code extension provides the GraphQL Try it view, which gives the ability to try GraphQL services within VS Code instead of using any third-party software.
```